prolog/tokenize.pl

```diff
@@ -231,22 +238,27 @@
 
 pack(X, Count) --> [X], pack(X, 1, Count).
 
-pack(_, Total, Total)      --> call(eos).
+pack(_, Total, Total)      --> eos.
 pack(X, Total, Total), [Y] --> [Y], { Y \= X }.
 pack(X, Count, Total)      --> [X], { succ(Count, NewCount) },
                                pack(X, NewCount, Total).
 
 
 
-% PARSING
+		 /*******************************
+		 *        PARSING               *
+		 *******************************/
 
-tokens([T])    --> token(T), call(eos), !.
+
+tokens([T])    --> token(T), eos, !.
 tokens([T|Ts]) --> token(T), tokens(Ts).
 
 % NOTE for debugging
 % tokens(_)   --> {length(L, 200)}, L, {format(L)}, halt, !.
 
-token(word(W))     --> word(W), call(eos), !.
+token(number(N))   -->
+    number(N), !.
```

Comment:
Seems like this should stay on the same line, to fit the rest of the clauses (unless we want to reformat all the other clauses)?

Reply:
fixed by latest